src/Language/GraphQL/Draft/Parser.hs

```diff
@@ -280,9 +301,10 @@ typeDefinition =
 optDesc :: Parser (Maybe AST.Description)
 optDesc = optional (AST.Description <$> stringLiteral)
```

Comment:
For your information: there is a current shortcoming in how descriptions in schema documents are parsed, since `stringLiteral` only accepts single-quoted strings, whereas multiline strings are also supported (see #25 and #30).

Reply:
> only accepts single-quoted string
Do you mean double-quoted strings? Because, IIUC it's used to parse GraphQL string values also which are supposed to only be double quoted strings.

> multiline strings are also supported
Oh, I didn't know about this. I will add support for this. I prefer to do that in a separate PR so that it's not a blocker for this PR. Please let me know if you feel differently about this.

Reply:
Sorry, yes, I did mean double-quoted strings. What I meant to say was that they were quoted with a _single_ double-quote, as in `"hello world"`, as opposed to multiline strings surrounded in three double quotes, as in
```
"""
hello there!
this is a string
"""
```

Reply:
Anyway, yes, I agree that this is not a blocker, I just thought it was perhaps relevant to what you're working on.